### ISSUE TITLE
Deploying RL agents without the training environment

### ISSUE THREAD

**Issue by ayansengupta17** [tensorflow/agents#387]
I want to deploy a trained RL agent in a practical field for testing. Usually with saved policy, we need to provide Time_step to get the next action. It is actually just a named tuple TimeStep(step_type, reward, discount, observation). 

So when I don't have an environment, but in the real field, I will only have the observation space. How can I use it with the saved policy? Do I need to manually create the named tuple with all tf,constants ?

**Comment by summer-yue** [comment]
Sorry about the delay. Sounds like you have a pretrained saved model (trained on some simulated environment similar to your practical environment with proper time steps), and you want to try to deploy it in your environment?

If that's the case, you can convert the observations and rewards to a time step. That should be straight forward. You can manually create the tuple, with the same format as the time steps used by the pretrained model. Let us know if that answers your question.

**Comment by ayansengupta17** [comment]
Thanks @summer-yue. I exactly did the same thing and its working. I was just wondering if tf-agents had some similar in-built functions.

**Comment by sguada** [comment]
We have some functions that create TimeSteps given observations, such as [time_step.restart](https://github.com/tensorflow/agents/blob/master/tf_agents/trajectories/time_step.py#L108) at the beginning of the episode, [time_step.transition](https://github.com/tensorflow/agents/blob/master/tf_agents/trajectories/time_step.py#L178) for the middle transitions and [time_step.termination](https://github.com/tensorflow/agents/blob/master/tf_agents/trajectories/time_step.py#L245) to terminate the episode.

You can also pass `batch_size=1`
